### DISCUSSION TITLE
How to track which printed QR code was used?

### DISCUSSION THREAD

**Discussion by iBobik** [btcpayserver/btcpayserver#3655] [Technical Support] [answered]
We use printed QR code (from POS app), one code for one branch. I expected that there will be info which QR code (title or description of the item) was used in the invoice, but it is not like that.

How to track what QR code is responsible for specific payment? (Without having different stores.)

**Comment by dennisreimann** [comment, marked as answer]
I might add this to #3667 tomorrow. Have the code up already, need to test it a bit more.

**Reply by dennisreimann** [reply]
Added it in the mentioned PR: With this change the LNURL meta data will be unified with the Lightning invoice meta data.
